test/unit/StakingProducts/initializeProducts.js

```diff
@@ -72,8 +102,54 @@ describe('initializeProducts', function () {
     await expect(stakingProducts.setInitialProducts(poolId, [...validProducts])).to.not.be.reverted;
   });
 
+  it('should initialize 1000 products with 2 weight', async function () {
```

Comment:
this test name is a bit hard to understand "with 2 weight"

shouldn't it be "with weight = 2"